spec/unit/rom/support/inflector_spec.rb

```diff
@@ -52,4 +56,27 @@
     end
     it_behaves_like 'an inflector'
   end
+
+  context 'without an inflector library' do
+    around do |example|
+      begin
+        api.remove_instance_variable(:@inflector)
+        original_load_path = $LOAD_PATH.dup
+        # Remove inflectors from the load path
+        Gem::Specification.each do |g|
+          if g.name == 'activesupport' || g.name == 'inflecto'
+            $LOAD_PATH.delete(g.lib_dirs_glob)
+          end
+        end
+        example.run
+      ensure
+        # restore the load path
+        $LOAD_PATH.clear.concat(original_load_path)
+        api.setup
+      end
+    end
```

Comment:
Let me know if you need help with this. It'd be awesome to include that patch in 0.6.0 :)